ENTITY/ApplyForceToEntity.md

```diff
@@ -5,78 +5,62 @@ ns: ENTITY
 
 ```c
 // 0xC5F68BE9613E2D18 0xC1C0855A
-void APPLY_FORCE_TO_ENTITY(Entity entity, int forceType, float x, float y, float z, float offX, float offY, float offZ, int boneIndex, BOOL isDirectionRel, BOOL ignoreUpVec, BOOL isForceRel, BOOL p12, BOOL p13);
+void APPLY_FORCE_TO_ENTITY(Entity entity, int forceType, float x, float y, float z, float offX, float offY, float offZ, int nComponent, BOOL bLocalForce, BOOL bLocalOffset, BOOL bScaleByMass, BOOL bPlayAudio, BOOL bScaleByTimeWarp);
 ```
 
-Applies a force to the specified entity.
+Apply a force to an entity.
 
-**List of force types (p1)**:
-
-```
-public enum ForceType
-{
-    MinForce = 0,
-    MaxForceRot = 1,
-    MinForce2 = 2,
-    MaxForceRot2 = 3,
-    ForceNoRot = 4,
-    ForceRotPlusForce = 5
+```cpp
+enum eCommandApplyForceTypes {
+    APPLY_TYPE_FORCE = 0,
+    APPLY_TYPE_IMPULSE = 1,
+    APPLY_TYPE_EXTERNAL_FORCE = 2,
+    APPLY_TYPE_EXTERNAL_IMPULSE = 3,
+    APPLY_TYPE_TORQUE = 4,
+    APPLY_TYPE_ANGULAR_IMPULSE = 5
 }
 ```
 
-Research/documentation on the gtaforums can be found [here](https://gtaforums.com/topic/885669-precisely-define-object-physics/) and [here](https://gtaforums.com/topic/887362-apply-forces-and-momentums-to-entityobject/).
```

Comment:
If these are both still active links with useful information, do we need to delete them? And if we are deleting them, are there anything in those 2 links that we could capture in the docs here that would be of use?

Reply:
true that, i figured the docs may explain most the things covered in those threads, added them back :)